src/components/Settings.jsx

```diff
@@ -97,10 +107,56 @@ export default function Settings() {
   const isSats = settings.unit === SATS
   const isLightTheme = settings.theme === window.JM.THEMES[0]
 
+  const lockWallet = useCallback(
+    async ({ force, destination }) => {
+      if (!force && (serviceInfo.coinjoinInProgress || serviceInfo.makerRunning)) {
+        setShowConfirmLockModal({ destination })
+        return
+      }
+
+      setLockingWallet(true)
+
+      try {
+        const { name: walletName, token } = currentWallet
+        const res = await Api.getWalletLock({ walletName, token })
+
+        if (res.ok || res.status === 401) {
+          stopWallet()
+        } else if (!res.ok) {
```

Comment:
```suggestion
        } else {
```
Is there any other possibility?